data/json/items/gun/9mm.json

```
@@ -190,6 +165,64 @@
     "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "BURST", "3 rd.", 3 ], [ "AUTO", "auto", 12 ] ],
     "built_in_mods": [ "mp5sd_suppressor" ]
   },
+  {
+    "id": "hk_mp5k",
+    "type": "GUN",
+    "copy-from": "hk_mp5",
+    "name": "H&K MP5K-PDW",
+    "description": "The Heckler & Koch MP5 is one of the most widely-used submachine guns in the world, and has been adopted by special police forces and militaries alike.  Its high degree of accuracy and low recoil are universally praised.  The MP5K-PDW model features a shorter barrel, a built in foregrip, and a side folding stock for vehicle or aircraft crew use.",
+    "volume": "1753 ml",
+    "bashing": 8,
+    "dispersion": 270,
+    "ranged_damage": 0,
+    "handling_modifier": 6,
+    "valid_mod_locations": [
+      [ "accessories", 3 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ]
+    ],
+    "flags": [ "NEEDS_UNFOLD", "COLLAPSIBLE_STOCK" ]
+  },
+  {
+    "id": "hk_mp5k",
```

ID collision with the MP5K-PDW.
```suggestion
    "id": "ptr603",
```

---

Yeah, sorry, I wanted to push this so I could work on it on another machine. I actually had work to do last night so didn't get around to it